install_config/topics/configuring_azure_authorization.adoc

```diff
@@ -0,0 +1,75 @@
+////
+Module included in the following assemblies:
+
+install_config/configuring_azure.adoc
+////
+
+[id='configuring-azure-permissions_{context}']
+= Configuring authorization for Microsoft Azure
+
+*Azure roles*
+
+Configuring Microsoft Azure for {product-title} requires the following role:
+
+[cols='1,3']
+|===
+
+|Contributor
+|To create and manage all types of Microsoft Azure resources.
+
+|===
+
+See the
+link:https://docs.microsoft.com/en-us/azure/role-based-access-control/rbac-and-directory-admin-roles[Classic
+subscription administrator roles vs. Azure RBAC roles vs. Azure AD administrator
+roles documentation] for more information.
+
+*Permissions*
+
+Configuring Microsoft Azure for {product-title} requires a service principal,
+which allows the creation and management of Kubernetes service
+load balancers and disks for persistent storage. The service principal values
+are defined at installation time and deployed to the Azure configuration file, which is `/etc/origin/cloudprovider/azure.conf` on {product-title} master and node hosts.
+
+.Procedure
+
+. Using the Azure CLI, obtain the account subscription ID:
++
+[subs=+quotes]
+----
+# az account list
+[
+{
+  "cloudName": "AzureCloud",
+  *"id": "<subscription>",*
```

Comment:
Are the asterisks added for emphasis? Can I delete them?

Reply:
You can delete the astrics. They were just there to bring it to light but fine without